Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:14
+
+EXPOSE 8079
+
+WORKDIR /usr/src/app
+
+COPY package.json .

```

Comment:
Remove this and just do one copy instruction, or combine the copy instructions into one
```suggestion
```